src/features/More/moreApi.ts

```diff
@@ -0,0 +1,13 @@
+import axios from "axios";
+
+const moreApi = {
+  getCategories: async () => {
+    const result = await axios.get(
```

Comment:
If you use await you MUST wrap it in a try-catch.
Please implement this change across the app